docs/fundamentals/accessibility.md

```diff
@@ -0,0 +1,378 @@
+> Semantic properties don't try to force equivalent behavior on each platform. Instead, they rely on the accessibility experience provided by each platform.
+
+The `SemanticProperties` class defines the following attached properties:
+
+- `Description`, of type `string`, which represents a description that will be read aloud by the screen reader. For more information, see [Description](#description).
+- `Hint`, of type `string`, which is similar to `Description`, but provides additional context such as the purpose of a control. For more information, see [Hint](#hint).
+- `HeadingLevel`, of type `SemanticHeadingLevel`, which enables an element to be marked as a heading to organize the UI and make it easier to navigate. For more information, see [Heading levels](#heading-levels).
+
+These attached properties set native accessibility values so that a screen reader can speak about the element. <!-- For more information about attached properties, see [Attached properties]().-->
+
+### Description
+
+The `SemanticProperties.Description` attached property represents a short, descriptive `string` that a screen reader uses to announce an element. This property should be set for elements that have a meaning that's important for understanding the content or interacting with the user interface. Setting this property can be accomplished in XAML:
+
+```xaml
+<Image Source="dotnet_bot.png"
+       SemanticProperties.Description="Image of dotnet bot." />
+```
+
+Alternatively, it can be set in C#:
+
+```csharp
+Image image = new Image();
+SemanticProperties.SetDescription(image, "Image of dotnet bot.");
+```
+
+In addition, the `SetValue` method can also be used to set the `SemanticProperties.Description` attached property:
+
+```csharp
+image.SetValue(SemanticProperties.DescriptionProperty, "Image of dotnet bot.");
+```
+
+> [!WARNING]
+> Don't set the `SemanticProperties.Description` attached property on a `Label`. This will stop the `Text` property being spoken by the screen reader. Instead, set the `Text` property, and the `SemanticProperties.Hint` attached property if required.
+
+The accessibility information for an element can also be defined on another element. For example, a `Label` next to an `Entry` can be used to describe what the `Entry` represents. This can be accomplished in XAML as follows:
+
+```xaml
+<Label x:Name="label"
+       Text="Enter your name: " />
+<Entry SemanticProperties.Description="{Binding Source={x:Reference label} Path=Text}" />
+```
+
+Alternatively, it can be set in C# as follows:
+
+```csharp
+Label label = new Label
+{
+    Text = "Enter your name: "
+};
+Entry entry = new Entry();
+SemanticProperties.SetDescription(entry, label.Text);
+```
+
+### Hint
+
+The `SemanticProperties.Hint` attached property represents a `string` that provides additional context to the `SemanticProperties.Description` attached property, such as the purpose of a control. Setting this property can be accomplished in XAML:
+
+```xaml
```

Comment:
I don't mind changing the example, but I don't see how the suggestion fits with the text about Hint providing additional context to Description. For clarity, that requires an example that sets Description and Hint. Not doing so will result in user confusion, and they will open issues. Thoughts?

Reply:
The description is the same as the text since it's a text-based control, but I can see how that can be unclear.
How about an image button that conveys liking something? The image can be a thumbs up or heart image, description is "Like", hint is "Likes this post"

Reply:
Perfect, thanks :)